jmeter-java-dsl-graphql/src/test/java/us/abstracta/jmeter/javadsl/graphql/DslGraphqlSamplerTest.java

```diff
@@ -137,6 +138,41 @@ public void shouldSendGraphqlQueryToServerWhenGraphqlSamplerWithHttpGet()
     ).run();
     verify(getRequestedFor(anyUrl()).withQueryParam("query", equalTo(QUERY)));
   }
+  @Test
+  public void shouldThrowExceptionForInvalidVariable() {
```

Comment:
This is a good test, thank you for adding it! Can you please change the name to match the convention we have followed for the rest (shouldXWhenY)? For example: shouldThrowIllegalArgumentWhenGraphqlSamplerWithInvalidVariable